src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/transpose_matmul.cpp

```diff
@@ -150,6 +194,7 @@ INSTANTIATE_TEST_SUITE_P(smoke_Snippets_DynMatMult, TransposeMatMul,
 //                                  ::testing::ValuesIn(transpose_input_shapes),
 //                                  ::testing::Values(2), // Transpose on Matmul output
 //                                  ::testing::Values(ov::element::i8),
+//                                  ::testing::Values(MatMulType::MatMul),
```

Comment:
I suggest to remove this commented out test instance (moreover, I'm author and I remember that we thought we will support it soon)

The same suggestion is for test instance above.

Reply:
Removed, thanks